src/components/AccountDrawer/MiniPortfolio/Tokens/index.tsx

```diff
@@ -33,27 +41,35 @@ export default function Tokens({ account }: { account: string }) {
 
   const { data } = useCachedPortfolioBalancesQuery({ account })
 
-  const visibleTokens = useMemo(() => {
-    return !hideSmallBalances
-      ? data?.portfolios?.[0].tokenBalances ?? []
-      : data?.portfolios?.[0].tokenBalances?.filter((tokenBalance) =>
-          meetsThreshold(tokenBalance, hideSmallBalances)
-        ) ?? []
-  }, [data?.portfolios, hideSmallBalances])
-
-  const hiddenTokens = useMemo(() => {
-    return !hideSmallBalances
-      ? []
-      : data?.portfolios?.[0].tokenBalances?.filter(
-          (tokenBalance) => !meetsThreshold(tokenBalance, hideSmallBalances)
-        ) ?? []
-  }, [data?.portfolios, hideSmallBalances])
+  // eslint-disable-next-line react-hooks/exhaustive-deps
+  const tokenBalances = data?.portfolios?.[0].tokenBalances ?? []
+  type TokenBalance = (typeof tokenBalances)[0]
+
+  const [visibleTokens, hiddenTokens] = useMemo(() => {
```

Comment:
Can we add some tests for spam/non-spam/<$1/>$1/no value tokens in the MiniPort?